### PR TITLE
Add using batch create for the inventories - seconds to create 1000 records instead of minutes

### PR DESCRIPTION
**Pull request by asaf-erlich**
# Description of change
Every 3 days it has to recreate 1000+ records, which took us minutes cause it repeats many calls (like get all locations) and also calls create or batch for 1 record * 1000, instead of batching it. This uses the batch apis correct.

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tests/test_client.py [hunk 1]
```diff
@@ -214,68 +215,75 @@ def get_an_inventory_adjustment(self, obj_id):
             raise RuntimeError('GET INVENTORY_ADJUSTMENT: {}'.format(response.errors))
         return response
 
-    def create_inventory_adjustment(self, start_date=None):
+    @staticmethod
+    def _chunks(lst, n):
```
Comment (luandy64) [inline tests/test_client.py]: We should talk to QA, but I think we can import this from the tap

File: tests/test_client.py [hunk 2]
```diff
@@ -214,68 +215,75 @@ def get_an_inventory_adjustment(self, obj_id):
...
+        """Yield successive n-sized chunks from lst."""
+        for i in range(0, len(lst), n):
+            yield lst[i:i + n]
+
+    def create_batch_inventory_adjustment(self, num_records):
         # Create an item
-        item = self.create_item().body.get('objects')[0]
+        #item = self.create_item().body.get('objects')[0]
```
Comment (luandy64) [inline tests/test_client.py]: Let's delete this line at some point

File: tests/test_client.py [hunk 3]
```diff
@@ -408,61 +393,43 @@ def create_modifier_list(self):
                                                                       'ordinal': 1}
                                                                  ],}
                                           }]}],
+
+    def create_item(self, num_records=1):
+        start_date = '2020-07-29T00:00:00Z'
```
Comment (luandy64) [inline tests/test_client.py]: I think we said this, but I want to note it: we'll change this to use the start date in the test rather than be hard coded

**Review by luandy64 — APPROVED** [review]

File: tests/test_client.py (hunk 1 above)
Comment (asaf-erlich) [reply tests/test_client.py]: Did this, thanks

File: tests/test_client.py (hunk 2 above)
Comment (asaf-erlich) [reply tests/test_client.py]: Done

File: tests/test_client.py (hunk 3 above)
Comment (asaf-erlich) [reply tests/test_client.py:398]: ➕

**Comment by asaf-erlich** [comment]
Pylint passes, tests are running, but fail in some asserts. Will fix those in a subsequent pr.

**Pull request merged**